electron/createWindow.js

```diff
@@ -126,9 +128,13 @@ export default appState => {
   });
 
   window.once('ready-to-show', () => {
-    window.show();
+    startMinimized ? window.hide() : window.show();
   });
 
+  window.webContents.once('dom-ready', () => {
+    startMinimized && window.hide()
```

Comment:
Is this needed? I would imagine once the `dom-ready` event is fired the window should already be hidden.

Reply:
It seemed like this is the one that makes it work on windows -reliably-.

Reply:
Should this line be just removed https://github.com/lbryio/lbry-desktop/pull/3236/files#diff-dd110088e25f2da03c36e833bd1295a8R131

And placed here instead?